scipy/interpolate/fitpack2.py

```diff
@@ -83,23 +83,23 @@ class UnivariateSpline:
     y : (N,) array_like
         1-D array of dependent input data, of the same length as `x`.
     w : (N,) array_like, optional
-        Weights for spline fitting.  Must be positive.  If None (default),
-        weights are all equal.
+        Weights for spline fitting.  Must be positive.  If `w` is None,
+        weights are all equal. Default is None.
     bbox : (2,) array_like, optional
         2-sequence specifying the boundary of the approximation interval. If
-        None (default), ``bbox=[x[0], x[-1]]``.
+        bbox is None, ``bbox=[x[0], x[-1]]``. Default is None.
     k : int, optional
         Degree of the smoothing spline.  Must be 1 <= `k` <= 5.
-        Default is `k` = 3, a cubic spline.
+        `k` = 3 is a cubic spline. Default is 3.
```

Comment:
>  In this context, the inheritance of UnivariateSpline also cause an issue in:
>  ...
> Can it be fixed in the same PR or should be open a new issue?

What do you mean by `cause an issue`? I guess it would be better to do another PR as here you are addressing a given issue so it might be out of this framework. Branches are free 😉